app/components/Threads/index.jsx

```diff
@@ -20,29 +13,52 @@ export default class ThreadsSection extends React.Component {
     local: PropTypes.object.isRequired,
   }
 
+
   constructor(props) {
     super(props);
     this.state = {
-      currentTab: peopleTab.id,
+      currentTab: 'Channels',
     };
   }
 
 
-  changeTab = (tab) => {
-    this.setState({ currentTab: tab.id });
+  changeTab = (tabName) => {
+    this.setState({
+      currentTab: tabName,
+    });
   };
 
+
   render() {
     const {channels, users, setCurrentChannel, local} = this.props;
+
+    const tabs = [
+      {
+        name: 'People',
+        list: users,
+      },
+      {
+        name: 'Channels',
+        list: channels,
+      },
+    ];
+
+    const currentTabData = tabs.filter(tab => tab.name === this.state.currentTab)[0];
```

Comment:
We can use Array.prototype.find from ES6:
tabs.find(tab =>tab.name === this.state.currentTab)